### ISSUE TITLE
What is your plan for next release?

### ISSUE THREAD

**Issue by zargarzadehm** [bnb-chain/tss-lib#242]
According to [this article](https://medium.com/@iofinnet/security-disclosure-for-ecdsa-and-eddsa-threshold-signature-schemes-4e969af7155b) and these CVEs - [CVE-2022–47930](https://www.cve.org/CVERecord?id=CVE-2022-47930), [CVE-2022–47931](https://www.cve.org/CVERecord?id=CVE-2022-47931), [CVE-2023–26556](https://www.cve.org/CVERecord?id=CVE-2023-26556), and [CVE-2023–26557](https://www.cve.org/CVERecord?id=CVE-2023-26557) - some vulnerabilities have been found in the protocol and in this package. I found [PR #233](https://github.com/bnb-chain/tss-lib/pull/233) related to 'Collision of Hash Values', but a new release has not been made yet. Can you tell me what your plan is for the next release, including this PR? Also, are there any side effects for other vulnerabilities in this library? If yes, do you have any plans to fix them and release a new version?

**Comment by DecoratedWings** [comment]
I have the same question as I stumbled upon the medium article in my analysis. Can a core maintainer kindly address this?

**Comment by nann-cheng** [comment]
Yes, still after 2 years, looking into the current release v2.0.2:

* CVE-2022–47930: Replay Attacks Involving Proofs
* CVE-2023–26556: Golang’s big.Int Arithmetic Not Constant-Time
* CVE-2023–26557: Non-Constant Time Scalar Multiplication For secp256k1

These three issues are still there, be cautious using it!
